event-notification/src/main/kotlin/com/d3/notification/controller/NotificationController.kt

```diff
@@ -41,13 +41,17 @@ class NotificationController(
         logger.info("New subscriber. Account id $accountId. Client id $clientID")
         return Flux.create { emitter ->
             val disposable = notificationListener.subscribeWithdrawalProofs { soraWithdrawalProofEvent ->
-                if (soraWithdrawalProofEvent.accountIdToNotify == accountId) {
-                    val event = ServerSentEvent.builder<String>()
-                        .event("sora-withdrawal-proofs-event")
-                        .data(gson.toJson(soraWithdrawalProofEvent))
-                        .build()
-                    logger.info("Publish to client $clientID. Event $soraWithdrawalProofEvent")
-                    emitter.next(event)
+                try {
+                    if (soraWithdrawalProofEvent.accountIdToNotify == accountId) {
+                        val event = ServerSentEvent.builder<String>()
+                            .event("sora-withdrawal-proofs-event")
+                            .data(gson.toJson(soraWithdrawalProofEvent))
+                            .build()
+                        logger.info("Publish to client $clientID. Event $soraWithdrawalProofEvent")
+                        emitter.next(event)
+                    }
+                } catch (e: Exception) {
+                    logger.error("Cannot stream events via SSE for client $clientID", e)
```

Comment:
whats next, any nacks, deaths etc?

Reply:
Oh, no. There is no need to nack or ack. It's a client's side error.